packages/wdio-reporter/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "rootDir": "./src"
   },
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "tests/**/*"
```

Comment:
@ablok tests should not be included here, we don't want to build and ship them 😉

Reply:
They have to be, otherwise you will not get any typescript errors.

Reply:
We should get typescript errors if you run Jest tests. This is the experience I made with other packages too.

Reply:
I want errors when I am developing something. Fast feedback is essential.

Reply:
I agree on the feedback, you don't want to be surprised with errors after running tests. We can whitelist/blacklist directories when publishing, we can use the tsconfig or the .npmrc for that I think while meantaining the feedback during development.

Reply:
> I want errors when I am developing tests. Fast feedback is essential.

I totally agree. I however get type errors in my VSCode in my tests without including tests. Looking at other setups (e.g. Jest I don't think we need include at all).

Reply:
Hmm, that's weird. This is the only way I could get it to show errors. I don't know what's wrong.

Reply:
Removed the folder, but there must be something wrong with the project setup. Or am I the only one that is not getting typescript errors without it?